lib/src/cartesian/components/AxisImage.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import {
+  Image,
+  useImage,
+  type ImageProps,
+  type SkImage,
+} from "@shopify/react-native-skia";
+
+export type TickImage = Partial<Omit<ImageProps, "image">> & {
+  image?: string;
+  skImage?: SkImage | null;
+  width?: number;
+  height?: number;
+};
+
+type AxisImageProps = TickImage & {
+  y: number;
+  x: number;
+};
+
+export const AxisImage: React.FC<AxisImageProps> = ({
+  image,
+  skImage,
+  y,
+  x,
+  ...rest
+}) => {
+  const imageSKFromHook = useImage(image || "");
+  const imageSK = skImage || imageSKFromHook;
+
+  return (
+    <Image
+      image={imageSK}
+      fit="contain"
+      y={y}
+      x={x}
+      width={12}
+      height={12}
+      {...rest}
```

Comment:
just confirming, `width` and `height` _can_ be overridden by `rest` here?

Reply:
Ah noted, well I think if we go with the alternative approach in the last commit of `render*Label`, we wouldn't include this then anyways, and leave it to users to handle rendering an image themselves.

Reply:
I wasn't sure if this would be kept as a convenience component for people to use?

Reply:
Nah, not worth it, but I've copied it into the examples area and people could just copy it from there.